src/portuguese.js

```diff
@@ -0,0 +1,43 @@
+
+class PresenteIndicativo {
+  conjugate (word, info) {
+    let ending = [];
+    switch (word.slice(-2)) {
+      case 'ar':
+        ending = info.singular ? ['o', 'as', 'a'] : ending = ['amos', 'ais', 'am'];
+        break;
+      case 'er':
+        ending = info.singular ? ['o', 'es', 'e'] : ['emos', 'eis', 'em'];
```

Comment:
In my opinion, info should not have a singular option, but subject string instead. It should be either of these: eu, tu, voce, ele, ela, vos, voces, etc. From the subject we can tell if it is singular or necessary, so info.singular is unnecessary right?